cli/pcluster/config/mappings.py

```diff
@@ -634,12 +635,12 @@
         ("enable_efa", {
             "type": BooleanJsonParam,
             "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP,
-            "default": False
+            "validators": [queue_xor_cluster_params_validator]
```

Comment:
You removed the default, is it right?

Reply:
Yes, there should be no default as the value now is taken from cluster section if not specified here. The ``queue_xor_cluster_params`` validator checks that the value is set here or there (but not in both sections).